iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/ParameterDeclarationImpl.java

```diff
@@ -44,7 +45,7 @@
 
 import static org.sonar.iac.arm.tree.ArmHelper.addChildrenIfPresent;
 
-public class ParameterDeclarationImpl extends AbstractArmTreeImpl implements ParameterDeclaration, HasDecorators {
+public class ParameterDeclarationImpl extends AbstractArmTreeImpl implements ParameterDeclaration, HasDecorators, HasKeyword {
```

Comment:
Same

Reply:
Same, should we return `null` on the json variant?